shared/charts/gpii-flowmanager/templates/service_entry_googleapis.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.enableStackdriverTrace }}
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  namespace: {{ .Release.Namespace | quote }}
+  name: {{ template "flowmanager.name" . }}-googleapis
+spec:
+  hosts:
+  - www.googleapis.com
```

Comment:
Why do we need access to `www.googleapis.com`? Also would it make sense to merge this in one `ServiceEntry` such as:
```yaml
  hosts:
  - www.googleapis.com
  - cloudtrace.googleapis.com
```
?

Reply:
Looks like when SA credentials provided, access to `www.googleapis.com` is not needed anymore.